Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2680ScrollView.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Linq;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.UITest; 
+using NUnit.Framework; 
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 2680, "[Enhancement] Add VerticalScrollMode/HorizontalScrollMode to ListView and ScrollView", PlatformAffected.All)]
+	public class Issue2680ScrollView : TestContentPage // or TestMasterDetailPage, etc ... 
+	{
+		protected override void Init()
+		{
+			// Initialize ui here instead of ctor 
+			var longStackLayout = new StackLayout();
+			Enumerable.Range(1, 50).Select(i => new Label() { Text = $"Test label {i}" })
+				.ForEach(label => longStackLayout.Children.Add(label));
+
+			var scrollView = new ScrollView
+			{
+				Orientation = ScrollOrientation.Vertical,
+				Content = longStackLayout
+			};
+
+			Content = scrollView;
+
+			scrollView.Orientation = ScrollOrientation.Neither;
+		}
+
+#if UITEST
+		[Test] 
+		public void Issue2680Test() 
+		{ 
```

Comment:
Can we add another test to show changing between the orientation like @samhouts said?

Reply:
Yes, that will help a lot!

Reply:
@rmarinho I've added ui-test in 52ba97934bd783c9a34be9e828350989ac1d0eee